core/src/main/java/org/apache/accumulo/core/client/admin/TableOperations.java

```diff
@@ -176,11 +176,27 @@ void create(String tableName, NewTableConfiguration ntc)
    *
    * @param tableName
    *          Name of a table to create and import into.
-   * @param importDir
-   *          Directory that contains the files copied by distcp from exportTable
+   * @param importDirs
+   *          A comma-delimited list of directories containing the files copied by distcp from
+   *          exportTable
    * @since 1.5.0
+   *
+   * @deprecated since 2.1.0 use {@link #importTable(String, Set)} instead.
+   */
+  @Deprecated
```

Comment:
This does not have to be deprecated because a new method was added.  Also, do you want the method to be removed in the future?  Starting with Java 9, two parameters were added to the deprecated annotation `since` and `forRemoval`.  The default for `forRemoval` is false.  If deprecating, it may also be good explicitly set these.  Not sure if my syntax is correct below, I have not used these yet.

```suggestion
  @Deprecated(since="2.1.0",forRemoval=false)
```

Personally, I don't think the method needs to be deprecated.

Reply:
I don't care whether it is deprecated or not, but would prefer the original method keep the behavior from before #1060 ; I think it's fine if not.

Reply:
> This does not have to be deprecated because a new method was added. 

Hmm, I don't understand why adding a new method eliminates the need for deprecation. The idea here would be to remove the original `importTable(String, String)` at some point when it is safe to do so.

Thanks for mentioning the Java 9 parameters. As such, I suspect the right annotation would be:
```
@Deprecated(since="2.1.0",forRemoval=true)
```

Reply:
> Hmm, I don't understand why adding a new method eliminates the need for deprecation.

The deprecation has nothing to do with the addition of the new method. It pertains to the removal of the old method.

> The idea here would be to remove the original `importTable(String, String)` at some point when it is safe to do so.

If you think it's better to remove the old method, that's fine and it should be deprecated. It's just not a requirement to remove it.

> @Deprecated(since="2.1.0",forRemoval=true)

If it is going to be deprecated, I agree it should be marked this way. I don't think we ever want to deprecate something and still retain it forever. The reason we deprecate stuff is so we can purge it (eventually). I wish `forRemoval=true` were default when they added this, because that's how I've always interpreted `@Deprecated`, but I recognize others may interpret it differently.

Reply:
> Hmm, I don't understand why adding a new method eliminates the need for deprecation.

I was just thinking that adding the new method does not necessitate deprecating the existing one.  The existing method is not broken in any way, it still satisfies its original functionality for anyone who invested the time to write code against it in the past.

Reply:
> I was just thinking that adding the new method does not necessitate deprecating the existing one. The existing method is not broken in any way, it still satisfies its original functionality for anyone who invested the time to write code against it in the past.

I'm happy to go either way. I suppose I'm in favor of keeping the number of public api methods to a minimum, but I don't feel strongly about leaving the existing method around if that's a better fit for the project.

Reply:
>  I suppose I'm in favor of keeping the number of public api methods to a minimum

That is a good goal, not sure whats best in the big scheme of things.   In my experience removing methods is most painful for indirect users of Accumulo (like someone using Accumulo via something like Rya).  Can get into a situation where you can not use the latest version of Accumulo w/o patching and rebuilding Rya. 

Could minimize clutter in the Accumulo source using a default method in the interface for the existing method (that calls the new method) and have it nowhere else.

Reply:
>That is a good goal, not sure whats best in the big scheme of things.

Ok, thanks for explaining your perspective. The makes a great deal of sense. Based on this, I'm going to avoid deprecating the original method. It looks like theres precedent for `default` implementations in the `TableOperations` interface, so I'll proceed in that way.